### PR TITLE
Handle client connect events asynchronously

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

Change-type: major|minor|patch 
Connects-to: # 
See: <url> 
Depends-on: <url> 

---
##### Contributor checklist

- [ ] Build output has been rebuilt and tested
- [ ] Introduces security considerations
- [ ] Tests are included
- [ ] Documentation is added or changed
- [ ] Affects the development, build or deployment processes of the component

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

**Comment by Page-** [comment]
@balena-ci I self-certify!

**Comment by Page-** [comment]
@balena-ci republish

**Comment by Page-** [comment]
@balena-ci republish

**Comment by Page-** [comment]
@balena-ci republish

File: src/vpn-worker.ts [hunk 1]
```diff
@@ -205,14 +207,23 @@ const worker = async (instanceId: number, serviceId: number) => {
...
-			// We need to flatten the uuid because otherwise it's a sliced string and keeps the
-			// original, rather large, string in memory forever
-			uuid: flatstr(data.common_name),
+			uuid,
 			bytes_received: 0,
 			bytes_sent: 0,
 			ts: process.hrtime()[0],
 		};
+
+		request.post({
```
Comment (thgreasi) [inline src/vpn-worker.ts:220]: Should we also have a `.catch` (+ report ?) so that the process doesn't crash if this request fails?

**Pull request merged**